lib/mpool/src/mpool.c

```diff
@@ -7,6 +7,11 @@
 
 #include <limits.h>
 #include <bsd/string.h>
+#include <sys/vfs.h>
+
+#if __linux__
+#include <linux/magic.h>
```

Comment:
Magical :)

Reply:
What is in this header? Never heard of it.

Reply:
It contains the superblock magic of the filesystems in Linux, I think...